src/core/qgsproperty.h

```diff
@@ -125,25 +127,57 @@ class CORE_EXPORT QgsPropertyDefinition
      * \param description can be any localised string describing what the property is used for.
      * \param helpText parameter should specify a descriptive string for users outlining the types
      * of value acceptable by the property (eg 'dashed' or 'solid' for a line style property).
+     * \param origin The origin of the property
+     * \param comment A free comment for the property
      */
-    QgsPropertyDefinition( const QString &name, DataType dataType, const QString &description, const QString &helpText );
+    QgsPropertyDefinition( const QString &name, DataType dataType, const QString &description, const QString &helpText, const QString &origin = QString(), const QString &comment = QString() );
 
     /**
      * Returns the name of the property. This is used internally and should be a unique, alphanumeric string.
      */
     QString name() const { return mName; }
 
     /**
+     * Sets the name of the property
```

Comment:
Is the `\since` test still not working?

Reply:
The test only works for classes, not members. And in any case this class was added in 3.0, so the \since isn't required here.

Reply:
Oh, I always thought it works on member level and wondered how the test finds out if it was added in a different version than the class (somehow would need access to the current branches target version)